wiki/howToUseSemantics.md

```diff
@@ -0,0 +1,76 @@
+<div id="main_content_wrap" class="outer">
+
+<section id="main_content" class="inner">
+
+## How to use semantics
+
+To import the project user need to add to pom.xml
+
+<div>
+
+    <repositories>
+       <repository>
+           <id>jitpack.io</id>
+           <url>https://jitpack.io</url>
+       </repository>
+    </repositories>
+    <dependency>
+        <groupId>com.github.Ericsson</groupId>
+        <artifactId>eiffel-remrem-semantics</artifactId>
+        <version>x.x.x</version>
+    </dependency>
+
+   </div>
+
+Or import the project to pom.xml  
+
+<div>
+
+    repositories {
+         ...
+         maven { url 'https://jitpack.io' }
+    }
+    dependencies {
+        compile("com.github.Ericsson:eiffel-remrem-semantics:x.x.x")
+    }
+
+   </div>
+
+<div>
+Eiffel REMReM Semantics contains implementation of Eiffel REMReM Protocol Interface. All methods of interface MsgService are implemented by SemanticsService class: generateMsg, getEventId, getEventType, getSupportedEventTypes, getEventTemplate, getServiceName, validateMsg, generateRoutingKey. Also in this class could be implemented more methods depending on the needs of developer.
+
+The main task of Eiffel Semantics is to check a correctness of Eiffel message. Project contains JSON schemas for each Eiffel type of event. If you need to create new event type, you need to define the JSON schema for this event.
```

Comment:
Remove the extra space before the sentence.

Reply:
removed extra spaces and rephrased the sentence